### PR TITLE
Tweak conv-rnn model

### PR DESCRIPTION
**Pull request by daemon**
- Fix misplaced zero_grad()
- Tweak model hyperparams and optimization algorithm

### CODE REVIEW

File: conv_rnn/test.py [hunk 1]
```diff
@@ -17,19 +18,19 @@ def main():
     parser.add_argument("--gpu_number", default=0, type=int)
     args = parser.parse_args()
 
-    model.set_seed(5, no_cuda=args.no_cuda)
-    data_loader = data.SSTDataLoader(args.data_dir)
+    model.set_seed(3, no_cuda=args.no_cuda)
```
Comment (tuzhucheng) [inline conv_rnn/test.py]: make seed configurable?

File: conv_rnn/model.py [hunk 2]
```diff
@@ -34,8 +34,6 @@ def __init__(self, word_model, **config):
         else:
             raise ValueError("RNN type must be one of LSTM or GRU")
         self.conv = nn.Conv2d(1, n_fmaps, (1, self.hidden_size * 2))
-        if dropout:
```
Comment (tuzhucheng) [inline conv_rnn/model.py]: does this hurt performance? if we are removing it why are we adding `--dropout_prob` to `train.py`?

Comment (daemon) [reply conv_rnn/model.py]: Paper says no dropout for SST1/2.

File: conv_rnn/test.py (hunk 1 above)
Comment (daemon) [reply conv_rnn/test.py]: Actually the seed is useless in test.py, I'll remove it later.
